far/FarEng.hlf.m4

```diff
@@ -6187,16 +6199,37 @@ $ #far:config Viewer.F8CPs#
 
 @Panel.Tree.TurnOffCompletely
 $ #far:config Panel.Tree.TurnOffCompletely#
-    Отключает (если = True) режим работы с деревом каталогов в панелях.
+    If “true”, all folder tree operations are unavailable:
 
-    При отключении становятся недоступными для выбора режимы дерева в панелях, поиске каталога (Alt+F10),
-конфигурация дерева в настройках и выбор каталога из дерева в диалогах копирования/перемещения файлов.
-Также не обновляются файлы кеша дерева каталогов (даже если они уже созданы) при создании/удалении/переименовании
-каталогов.
+  - ~Tree panel~@TreePanel@ mode in ~left and right menus~@LeftRightMenu@,
+as well as toggle tree panel shortcut key #Ctrl+T#.
 
-    По умолчанию значение = True
+  - ~Find folder~@FindFolder@ panel command (#Alt+F10#).
 
-    Изменение этого параметра возможно через ~far:config~@FarConfig@
+  - Folder tree operations in ~copy, move and rename~@CopyFiles@
+dialog (#F10# #Alt+F10# #Shift+F10#).
+
+    Also, folder tree cache files, even if already exist, are not updated
+during folder create / delete / rename operations.
+
+    Default value: True
+
+    This parameter can be changed via ~far:config~@FarConfig@ only.
+
+@Panel.Tree.AutoChangeFolder
```

Comment:
These parameters are available in the UI, do we need to describe them again? Maybe just add a reference to the Tree Settings topic?

Reply:
Thank you for pointing this out. I added help topic aliases. Was it what you meant? Or you'd prefer to have dedicated topics containing only "see..." references?

Reply:
Please have a look [how Far searches for an appropriate topic](https://github.com/FarGroup/FarManager/blob/5f4eae4156116a0857f27305452c95313fbed007/far/config.cpp#L2287) when you press Shift+F1 in far:config.

For example, for the "Panel.Tree.AutoChangeFolder" key Far first will try a topic of the same name and then "Panel.TreeSettings".

What I mean if a parameter is not a some semi-secret far:config setting and already available in UI and already described in a help topic of that UI - we really don't need to describe it separately: it's harder to maintain and keep in sync two copies and it leads the user to the wrong direction.
Instead it's better to just let Far show the topic of the corresponding UI.

However, it's not always possible: for example, we don't have the "Panel.TreeSettings" topic, it's just "TreeSettings", so it won't be found automatically. Renaming topics to follow the schema might be not a good idea since they have been there for ages and some third party plugins might refer to them in their hlf files, so it's better to add an alias, e.g. `@Panel.TreeSettings=TreeSettings`.

Reply:
Got it! Thank you. Single alias to the UI page instead of two aliases (which I did in iteration 2) is certainly better. Will change in iteration 3.